CONTRIBUTING.md

```diff
@@ -33,6 +33,21 @@ Here is an overview of [how `@octokit/rest` works](HOW_IT_WORKS.md).
 * Be sure to tag any issues your pull request is taking care of / contributing to.
 	* Adding "Closes #123" to a pull request description will auto close the issue once the pull request is merged in.
 
+## Testing a pull request from github repo locally:
+
+* You can install a specific pull request of a module from github using the bellow command
+`npm install owner/reponame#pullrequestId/branchname`
+
+* For example to test https://github.com/octokit/rest.js/pull/792 locally run...
+`npm install octokit/rest.js#791/error-headers`
+
+`octokit/rest.js` is the `owner/repo` of the github repository and `791/error-headers` is the `pullrequestId/branchname`. The branch name would default to `master` if it is not present.
```

Comment:
I think with the above change, we can leave out this sentence?

Reply:
This is mostly an example while the above one shows the general command to use.